index.html

```diff
@@ -5,28 +5,50 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="css/style.css">
+    <script src="js/code.js"> </script>
     <title>Document</title>
 </head>
+
 <body>
-    <div id = "block-body">
-        <header>
-          <div class= "logo">
-              <a href="index.html">
-                  <span class = "use">USE</span>-<span class = "web">WEB</span>.com
-              </a>
-              <p>Lore ipsum</p>
-          </div>
-          <div  class = "top-menu"></div>
-          <div class = "clock-top-author"></div>  
-        </header>
-    </div>
-  <h1>Registration Form</h1>
-    <form  >
-        <h1> Enter login</h1>
-        <label for="" value > Login
-        <input type = "text" name="login"></label>
+<div class="sing-in">  
+    <form action="" method = "POST">
+        <h1>Форма входа</h1>
+            <label for="login-field">Ваш логин</label>
+            <input type="text" name="login" id="login-field" >
+            <br>
+            <br>
+            <label for="password-field">Ваш пароль</label>
+            <input type="password" name="password" id="password-field">
+            <br>
+            <br>
+            <input type =  "checkbox" checked >Запомнить
+            <button class= "log-in"> Войти</button>
+            <br>
+        <button type = "button" class="log-in"><a href="#win1">Зарегестрироваться</a></button>
+</div>
+<a href="#x" class="overlay" id="win1"></a>
+<div class="popup">
+    <form action="" class ="sing-up" method="POST">
```

Comment:
Nice idea with having sing-in and sign-up in the same view as a popup!

Reply:
That`s my best idea!